libp2p/autotls/service.nim

```diff
@@ -149,31 +166,34 @@ method issueCertificate(
   let dns01Challenge = await acmeClient.getChallenge(@[domain])
   let keyAuth = acmeClient.genKeyAuthorization(dns01Challenge.dns01.token)
   let strMultiaddresses: seq[string] = self.peerInfo.addrs.mapIt($it)
+  echo strMultiaddresses
   let payload = %*{"value": keyAuth, "addresses": strMultiaddresses}
   let registrationURL = parseUri("https://" & AutoTLSBroker & "/v1/_acme-challenge")
 
   trace "Sending challenge to AutoTLS broker"
+  # before this runs we need to make sure mutiaddresses are available
   let (bearer, response) =
     await self.brokerClient.send(registrationURL, self.peerInfo, payload, self.bearer)
+  debug "Broker response", status = response.status
+  if response.status != HttpOk:
+    error "Failed to authenticate with AutoTLS Broker"
```

Comment:
Does the response contain any error msg in its body that could help figure out the reason of the error?